common/lib/xmodule/xmodule/assetstore/assetmgr.py

```diff
@@ -49,14 +50,10 @@ class AssetManager(object):
     @contract(asset_key='AssetKey', throw_on_not_found='bool', as_stream='bool')
     def find(asset_key, throw_on_not_found=True, as_stream=False):
         """
-        Finds a course asset either in the assetstore -or- in the deprecated contentstore.
+        Gets course asset in the deprecated contentstore.
+        This method was previously searching course asset either in the assetstore -or- in the deprecated contentstore.
+        We were able to not look into assetstore because we would raise AssetMetadataFoundTemporary.
+        It no longer finds it in assetstore because of multiple calls to unpickling compressed course structure
+        (because we removed calls to modulestore().find_asset_metadata(asset_key) which calls course structure look up)
```

Comment:
Please change this docstring to:
```
"""
Finds course asset in the deprecated contentstore.
This method was previously searching for the course asset in the assetstore first, then in the deprecated contentstore.
However, the asset was never found in the assetstore since an asset's metadata is not yet stored there.
The assetstore search was removed due to performance issues caused by each call unpickling the pickled and compressed
course structure from the structure cache.
"""
```